templates/zerver/api/add-reaction.md

```diff
@@ -9,6 +9,12 @@
 
 {generate_code_example(python)|/messages/{message_id}/reactions:post|example}
 
+{tab|js}
+
+More examples and documentation can be found [here](https://github.com/zulip/zulip-js).
```

Comment:
This line is getting repetitive; it might be a good candidate for a template in a later commit.

Reply:
On second thought, it might be getting a little bit circular since I'd prefer to point `zulip/zulip-js` to the main documentation instead. Let's just stop pointing to the zulip/zulip-js repo as the one containing documentation.

Reply:
@aero31aero sorry for late reply. So should I remove that line form all  the files.